src/middlewared/middlewared/plugins/config.py

```diff
@@ -37,6 +42,8 @@ class ConfigService(Service):
         `secretseed` will include the password secret seed in the bundle.
 
         `pool_keys` when set will include the geli encryption keys in the bundle.
+
+        `root_authorized_keys` when set will include "root" users "authorized_keys" file in the bundle.
```

Comment:
Just noticed `"root" users` also sounds strange. Perhaps ask docs review :P

Reply:
:P

Reply:
This is really unclear.  Is it supposed to be possessive?  In that case, avoid possessives, they are often unclear.  "When set" is not needed in any of these, and using present tense (include) is clearer than future tense (will include).  Can also factor out "in the bundle", like
```
Create a bundle of security-sensitive information. These options select which information is included in the bundle:

`secretseed`: include password secret seed.
`pool_keys`: include GELI encryption keys.
`root_authorized_keys`: include "authorized_keys" file for the root user.
```